assets/src/blocks/Cookies/CookiesBlock.js

```diff
@@ -1,13 +1,16 @@
 import { CookiesEditor } from './CookiesEditor.js';
+const { __ } = wp.i18n;
```

Comment:
Can we avoid adding this dependency to the front end? It adds a lot of scripts in order to perform the translation. In a few other blocks we added the translations to a back end render to avoid this.

Reply:
I guess we could just remove it tbh, it wasn't there before and nobody complained about it 😛 I've added it to be consistent with other blocks, but now that I think about it "Cookies" might be an international term and therefore not need translations 🤔 We could also fix this by splitting the blocks' files into backend and frontend scripts, but that would probably be better done in a separate PR?

Reply:
Sounds good, let's just remove the translation there.

Reply:
+1 on splitting back end and front end JS in a separate PR